members/B001267.yaml

```diff
@@ -55,7 +55,7 @@ contact_form:
         - name: Message
           selectors: [ "textarea#gen-fieldid-27" ]
           values: [ "$MESSAGE" ]
-          commands: [ "elements[0].value = elements[0].value.replace(/www\\.youtube\\.com\\/watch?[^&]*(&)?v=/gi, 'youtu.be/')" ]
```

Comment:
do we still need this?

Reply:
I don't think so. This was added to these yamls more than 6 months ago when a client added a youtube link to their advocacy message. I don't think is needed anymore for these targets